### PR TITLE
Family page: add new FAQ & Resource sections (fixes #12226)

### LINKED ISSUE #12226: Firefox for Families January 2023 Update
## Description

This is an update to the Firefox For Families landing page. 

[Details (including figma file, and copy document)](https://docs.google.com/document/d/1xPnRcn2H_7vheEyoGLkYu5zWqAeQkwVLz0s9GZVHcRE/edit?usp=sharing)

---

#### Success Criteria
- [x] Changes are staged and shared with PMM and Partnerships team
- [ ] Approval from PMM and Partnerships team

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Adds new FAQ & Resource sections

## Significant changes and points to review

Design and copy details (mozilla only): https://docs.google.com/document/d/1xPnRcn2H_7vheEyoGLkYu5zWqAeQkwVLz0s9GZVHcRE/edit
(note the checklist article in resource section has been replaced by latest parenting survey article, [see comment below](https://github.com/mozilla/bedrock/pull/12585#issuecomment-1409525992))

2 new sections & 2 new items in the subnav, replaced blurb content on Hero section to call out new FAQ content

New hero section text
_Ready to talk to your kids about internet stuff? Firefox is here for you. Bookmark this page and use the topics below to help get the conversation ball rolling. You got this!_

FAQ blurb in Hero links to FAQ section, pop in animates after title animation ends

FAQ section: on mobile, all details are closed by default, on larger screens, the first detail is open by default

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12226

## Testing

https://www-demo4.allizom.org/en-US/firefox/family/

To test this work:

- [x] http://localhost:8000/en-US/firefox/family/
- [x] http://localhost:8000/en-US/firefox/family/#faq (all links to Common Sense Media should have the following UTM params: `utm_source=firefox&utm_medium=partnership&utm_campaign=faq`)
- [x] http://localhost:8000/en-US/firefox/family/#resources

### CODE REVIEW

File: bedrock/firefox/templates/firefox/family/includes/resources.html
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/family/includes/resources.html:49]: note to self: lazy load

File: media/css/firefox/family/components/modules/_faq.scss
Comment (maureenlholland) [inline media/css/firefox/family/components/modules/_faq.scss:85]: note to self: include fallback

**Comment by maureenlholland** [comment]
⚠️ waiting on stakeholder feedback before continued development

File: media/css/firefox/family/components/modules/_faq.scss
Comment (maureenlholland) [inline media/css/firefox/family/components/modules/_faq.scss:97]: the hover color is not in protocol, so I'm using sass function to set the custom value from figma

**Comment by cherreramoz** [comment]
Summarizing notes from [Slack thread](https://mozilla.slack.com/archives/C037QQ265Q9/p1673966059639559):

**Design feedback:**
The updates look great! - the hover effect on the FAQ really helps on desktop and even on mobile to help call-out what question you're on. Couple comments:
- Can we time the pop-up for the CTA to happen after the "tech talk" pops in.
- For the CTA, our original thought was to have it be clickable and jump to the FAQ section. Hopefully that's something we can implement? If so; on hover, having the CTA grow in size slightly for the hover state.

**Copy feedback:**
Can we replace the intro copy at the top of the page (currently "Making awkward..") to:
_Ready to talk to your kids about internet stuff? Firefox is here for you. Bookmark this page and use the topics below to help get the conversation ball rolling. You got this!_

Thank you!

File: bedrock/firefox/templates/firefox/family/includes/modules/faq.html
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/family/includes/modules/faq.html:20]: **question for reviewer**: is it better to use the Details component here (with sections)? Not sure the HTML details semantics fit an FAQ section

File: media/css/firefox/family/components/_hero.scss
Comment (maureenlholland) [inline media/css/firefox/family/components/_hero.scss:56]: note: there might be some design changes here for the mobile behaviour of the CTA link

**Comment by cherreramoz** [comment]
@maureenlholland Hi! Sharing here for easier tracking:

We got one more edit request from the editorial team, which is not time-sensitive but can be done anytime:

In "More resources" a the bottom:

- Swap the 3rd feature (back-to-school checklist) with new content:

TITLE: Toddlers, tablets, and the "Tech Talk"
SUB-TEXT: We surveyed parents to get an understanding of how families use the internet across the U.S., Canada, France, Germany and the U.K.
URL: https://blog.mozilla.org/en/internet-culture/parenting-online-safety-report/
IMAGE: (https://blog.mozilla.org/wp-content/blogs.dir/278/files/2023/01/Parenting-Survey-1024x683.jpg)

**Comment by maureenlholland** [comment]
this is ready for code review

waiting on confirmation if the last commit (swapping an article in resources section) should be included or spun out into separate issue

File: media/css/firefox/family/components/_hero.scss
Comment (maureenlholland) [reply media/css/firefox/family/components/_hero.scss:56]: resolved to reduce the scale effect on mobile, we still see an outline on long taps but that appears to be mobile browser default behaviour and we are considering it non-blocking

File: media/css/firefox/family/components/_hero.scss
Comment (stephaniehobson) [inline media/css/firefox/family/components/_hero.scss:93]: This button is just a little too big at 360px and while there's no horizontal scroll bar everything ends up a bit off centre.
```suggestion
        min-width: 18ch;
```

File: media/css/firefox/family/components/modules/_faq.scss
Comment (stephaniehobson) [inline media/css/firefox/family/components/modules/_faq.scss:7]: Not a blocker: Why as `f3` and not as `utils`?

File: media/js/firefox/family/default-open-first-detail.es6.js
Comment (stephaniehobson) [inline media/js/firefox/family/default-open-first-detail.es6.js:21]: It's not open by default on desktop for me.

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
A couple small blockers and a couple nits. I'll admit I did not check the text against the copy provided since it seems like some other eyes have already been on that.

File: bedrock/firefox/templates/firefox/family/includes/modules/faq.html
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/family/includes/modules/faq.html:16]: It wasn't requested but doesn't hurt to have the option, will add

File: media/css/firefox/family/components/modules/_faq.scss
Comment (maureenlholland) [reply media/css/firefox/family/components/modules/_faq.scss:7]: Personal preference that stuck. From a folder view, `utils` filename works for me because it clearly belongs in the `firefox/family` folder, but from a single file view, `utils.$border-width` felt too generic as a namespace (could be utils from bedrock or firefox). So I wanted to call out that these variables and mixins are specifically from the Firefox for Families (F3) design.

File: bedrock/firefox/templates/firefox/family/includes/modules/faq.html
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/family/includes/modules/faq.html:14]: It's a good call-out. In this case, it would be a bigger change though, because I'm using these `c-card` class styles throughout the page. I'm really just borrowing the padding styling and white background on the details answers though, I can pull those out into their own class styles.

File: media/js/firefox/family/default-open-first-detail.es6.js
Comment (maureenlholland) [reply media/js/firefox/family/default-open-first-detail.es6.js:21]: I'm having trouble reproducing this. What browser/screen size is not opening? Any chance you have a reduced motion pref on?

File: bedrock/firefox/templates/firefox/family/includes/resources.html
Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/family/includes/resources.html:76]: should be fine, will update

Comment (maureenlholland) [reply bedrock/firefox/templates/firefox/family/includes/resources.html]: As I was looking at this, realized the copy doc says this should be "View more", which makes more sense because the resources are split between Youtube and the blog and this link only goes to the blog. Changed to "View more".

File: media/css/firefox/family/components/modules/_faq.scss
Comment (stephaniehobson) [reply media/css/firefox/family/components/modules/_faq.scss:7]: Ah, cool!

File: media/js/firefox/family/default-open-first-detail.es6.js
Comment (stephaniehobson) [reply media/js/firefox/family/default-open-first-detail.es6.js:21]: It's only a problem for me locally, it works fine on the demo. I'm using Nightly at 1260px (but widening the screen and refreshing did not fix it).

**Review by stephaniehobson — APPROVED** [review]
I'm still not having the details open for me locally but I don't know what the deadline is on  this and that's not a blocker so I'll R+ and leave it for @maureenlholland to merge

File: media/js/firefox/family/default-open-first-detail.es6.js
Comment (maureenlholland) [reply media/js/firefox/family/default-open-first-detail.es6.js:21]: Curiouser and curiouser. It's a nice-to-have and not a requirement, but never fun to discover a "sometimes" bug. I may file a follow-up issue if this is still popping up once merged to main.

**Pull request merged**
